src/transactions/transferFeedUtils.ts

```diff
@@ -400,3 +427,52 @@ export function isTransferTransaction(
 ): tx is TransferItemFragment {
   return (tx as TransferItemFragment).address !== undefined
 }
+
+// Note: This hook is tested from src/transactions/feed/TransferFeedItem.test.ts
+function useFiatConnectTransferDisplayInfo({ amount, transactionHash }: TokenTransfer) {
+  const { t } = useTranslation()
+  const tokenInfo = useTokenInfo(amount.tokenAddress)
+  const fcTransferDetails = useSelector(getCachedFiatConnectTransferSelector(transactionHash))
+  const cachedFiatAccountUses = useSelector(cachedFiatAccountUsesSelector)
+  const localCurrency = useSelector(getLocalCurrencyCode)
+  const account = useMemo(
+    () =>
+      cachedFiatAccountUses.find(
+        (account) => account.fiatAccountId === fcTransferDetails?.fiatAccountId
+      ),
+    [cachedFiatAccountUses, fcTransferDetails]
+  )
+  if (!account || !fcTransferDetails) {
+    return
+  }
+
+  const receive = Number(fcTransferDetails.quote.fiatAmount)
+  const total = Number(amount.value)
+  const exchangeRate = String(receive / total)
+  const sign = fcTransferDetails.quote.transferType === TransferType.TransferOut ? -1 : 1
```

Comment:
if you're not sure if the user is receiving or sending, it seems weird to call the variable assigned to fiat amount "receive"

Reply:
I'd probly just call it fiatAmount